Makefile.real

```diff
@@ -93,6 +93,8 @@ backport-include/backport/backport_path.h: .config Kconfig.versions Kconfig.kern
 		echo " * Changes will be overwritten"			;\
 		echo " */"						;\
 		echo ""							;\
+		echo "/* hack around predefined gcc/cpp "linux=1" */"	;\
```

Comment:
Can you add more description of the hack and/or reference of the problem

Reply:
You created a header file with:
```
#define BACKPORT_PATH /some/path/to/this/dkms/source/dir
```
The path is not quoted, and then when C preprocessor see this, it tries to map every token of the path to the macros already defined. "linux" is one of them that is very likely to appear since you're compiling a linux kernel dkms. It follows your path will be rewritten to a non-existent one, and everyone depending on the right **BACKPORT_PATH** will fail.